content/en/agent/proxy.md

```diff
@@ -400,6 +400,14 @@ frontend network_devices_netflow_frontend
     option tcplog
     default_backend datadog-network-devices-netflow
 
+# This declares the endpoint where your Agents connects for
+# receiving Remote Configurations (for example, the value of "remote_configuration.rc_dd_url")
```

Comment:
Just asking: do you want to add `#remove this section if your environment does not support Remote Configuration` here as well ?

Reply:
Removed it everywhere 👍🏻